include/ck/wrapper/utils/tensor_utils.hpp

```diff
@@ -0,0 +1,290 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2023, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include "ck/ck.hpp"
+
+#include "ck/utility/number.hpp"
+#include "ck/utility/tuple.hpp"
+#include "ck/utility/tuple_helper.hpp"
+#include "ck/utility/dynamic_buffer.hpp"
+#include "ck/utility/amd_address_space.hpp"
+
+namespace ck {
+namespace wrapper {
+
+// Disable from doxygen docs generation
+/// @cond
+// forward declarations
+template <typename Shape, typename Strides>
+struct Layout;
+template <AddressSpaceEnum BufferAddressSpace,
```

Comment:
You use `MemoryTypeEnum` and `AddressSpaceEnum` interchangeably, what's the idea? I know they are the same but wouldn't it be better to use only one of them so it's clear that it's indeed the same enum? If you introuduce the `MemoryTypeEnum` alias, maybe use only this one?

Reply:
Changed everywhere to MemoryTypeEnum